webapp/channels/src/actions/websocket_actions.jsx

```diff
@@ -239,6 +241,7 @@ export function reconnect() {
         }
 
         dispatch(fetchAllMyTeamsChannels());
+        dispatch(fetchTeamScheduledPosts(currentTeamId, true));
```

Comment:
This only resyncs for the current team. Will the other teams be resynced when we switch to them?

Reply:
yes, on every team switch, the scheduled messages information is resynced.